packages/formation-react/src/components/Table/usage.stories.mdx

```diff
@@ -0,0 +1,61 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Library/Table/Usage" />
```

Comment:
This doesn't actually have a `<Table>` in it - do we want to show designers what it looks like here?

Reply:
I thought about it, but adding that in here felt like a lot of boilerplate, and I figured they could go to the other story with the table in use—what's more, if they do that, they'll benefit from being able to actually control the viewport size with that storybook addon.

I'll add this as a task for the designer user research to prove or disprove my assumption.

Good call out. :+1: